projects/plugins/protect/src/js/components/unignore-threat-modal/index.jsx

```diff
@@ -0,0 +1,79 @@
+import { Button, getRedirectUrl, Text } from '@automattic/jetpack-components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Icon } from '@wordpress/icons';
+import { STORE_ID } from '../../state/store';
+import ThreatSeverityBadge from '../severity';
+import UserConnectionGate from '../user-connection-gate';
+import styles from './styles.module.scss';
+
+const UnignoreThreatModal = ( { id, title, label, icon, severity } ) => {
+	const { setModal, unignoreThreat } = useDispatch( STORE_ID );
+	const threatsUpdating = useSelect( select => select( STORE_ID ).getThreatsUpdating() );
+	const codeableURL = getRedirectUrl( 'jetpack-protect-codeable-referral' );
+
+	const handleCancelClick = () => {
+		return event => {
+			event.preventDefault();
+			setModal( { type: null } );
+		};
+	};
+
+	const handleUnignoreClick = () => {
+		return async event => {
+			event.preventDefault();
+			unignoreThreat( id, () => {
+				setModal( { type: null } );
+			} );
+		};
+	};
+
+	return (
+		<UserConnectionGate>
+			<Text variant="title-medium" mb={ 2 }>
+				{ __( 'Do you really want to unignore this threat?', 'jetpack-protect' ) }
+			</Text>
+			<Text mb={ 3 }>{ __( 'Jetpack will unignore the threat:', 'jetpack-protect' ) }</Text>
+
+			<div className={ styles.threat }>
+				<Icon icon={ icon } className={ styles.threat__icon } />
+				<div className={ styles.threat__summary }>
+					<Text className={ styles.threat__summary__label } mb={ 1 }>
+						{ label }
+					</Text>
+					<Text className={ styles.threat__summary__title }>{ title }</Text>
+				</div>
+				<div className={ styles.threat__severity }>
+					<ThreatSeverityBadge severity={ severity } />
+				</div>
+			</div>
+
+			<Text mb={ 4 }>
+				{ createInterpolateElement(
+					__(
+						'By choosing to unignore this threat, you acknowledge that you have reviewed the detected code. You are accepting the risks of maintaining a potentially malicious or vulnerable file on your site. If you are unsure, please request an estimate with <codeableLink>Codeable</codeableLink>.',
+						'jetpack-protect'
+					),
+					{
+						codeableLink: <Button variant="link" isExternalLink={ true } href={ codeableURL } />,
+					}
+				) }
+			</Text>
```

Comment:
```suggestion
```

What do you think about removing this blurb for the unignore step? Seems less relevant.

Reply:
Agreed!